main/ControllersPresenters/InterviewLevel.java

```diff
@@ -34,6 +34,20 @@ public String getInterviewOutput(String input) throws Exception{
             return getEndOfInterviewPrompt();
         }
         if (currentInterviewMaker.getChoiceOptions().contains(input)) {
+            //if choiceOptions is not the last thing the intern says, i.e. has already said response:
+            if (this.currentInterviewMaker.hasSaidResponse){
+                // GamePrompts.HIRE_INTERN;
+                // this.currentInterviewMaker;
+            }
+            //if choiceOptions is the last thing the intern says, i.e. is yet to say response:
+            if (! this.currentInterviewMaker.hasSaidResponse){
+                return GamePrompts.INTERN_RESPONSE_PROMPT +
+                        currentInterviewMaker.displayInternChoiceResponse(input) +
```

Comment:
So you see the logic here: If hasSaidResponse is false it goes here, but the method displayInternChoiceResponse will set hasSaidResponse to true. Fix this so that:
1. hasSaidResponse should be only set to true when the currentInterviewIntern has finished ALL of its responses.
2. it should be set back to false if you have another intern.
Since this will cause a lot of methods in InterviewLevel, you can move this part to interviewMaker if it's easier for you. That is: let interviewMaker check if an interviewee hasFinishedResponding and then return what is needed.